ktlint-ruleset-standard/src/main/kotlin/com/pinterest/ktlint/ruleset/standard/rules/ClassSignatureRule.kt

```diff
@@ -651,7 +651,7 @@ public class ClassSignatureRule :
     private fun List<ASTNode>.collectLeavesRecursively(): List<ASTNode> = flatMap { it.collectLeavesRecursively() }
 
     private fun ASTNode.collectLeavesRecursively(): List<ASTNode> =
-        if (psi is LeafElement) {
+        if (isLeaf()) {
```

Comment:
Yes, we would blow up this change too much if we would it include it as well. I just wanted to verify that it would make sense to replace the function with property + getter to improve readability. What do you think?